### PR TITLE
Typescript - Placeholder

### PR DESCRIPTION
**Pull request by ardalis**
Adding placeholder for later migration. This is a low priority but this work was already done and I want to get the branch closed out of my fork.

### CODE REVIEW

**Comment by danroth27** [comment]
Please update to use the stub notice and icon used by the other stub articles.

File: docs/client-side/typescript.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.. include:: /stub-topic.txt
+
+|stub-icon| Introducing TypeScript
+======================
+.. By :ref:`Noel Rice <typescript-author>` | Originally Published: 1 June 2015 
+
+.. TypeScript helps you build enterprise scale applications on the client with the same smooth development experience, reliability, and scalability that you expect from a .NET language. TypeScript lets you work with familiar abstractions like strong types, interfaces, classes, namespaces, enums, generics, and lambda expressions. TypeScript code editors have intelligent code completion (Intellisense in Visual Studio) and navigation between object definitions and their references. When you’re done coding, TypeScript compiles to pure JavaScript, leaving no TypeScript artifacts behind.
+
+.. (This article is in the process of being migrated from another format)
```
Comment (danroth27) [inline docs/client-side/typescript.rst]: Remove this line.

File: docs/client-side/typescript.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.. include:: /stub-topic.txt
+
+|stub-icon| Introducing TypeScript
+======================
```
Comment (danroth27) [inline docs/client-side/typescript.rst]: Fix heading underline to extend to the end of the title

File: docs/client-side/typescript.rst (hunk 1 above)
Comment (ardalis) [inline docs/client-side/typescript.rst]: done

File: docs/client-side/typescript.rst (hunk 2 above)
Comment (ardalis) [inline docs/client-side/typescript.rst]: done

**Comment by danroth27** [comment]
:shipit:

**Pull request merged**